Packages/InputSystem/Runtime/Interpreters/CallbackContextInterpreter.cs

```diff
@@ -0,0 +1,82 @@
+using UnityAtoms.BaseAtoms;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace UnityAtoms.InputSystem
+{
+    public class CallbackContextInterpreter<T, P, C, V, E1, E2, F, VI> : BaseAtom
+        where T : struct
+        where P : struct, IPair<T>
+        where C : AtomBaseVariable<T>
+        where V : AtomVariable<T, P, E1, E2, F>
+        where E1 : AtomEvent<T>
+        where E2 : AtomEvent<P>
+        where F : AtomFunction<T, T>
+        where VI : AtomVariableInstancer<V, P, T, E1, E2, F>
+    {
+        [SerializeField] private E1 _started;
+        [SerializeField] private E1 _performed;
+        [SerializeField] private E1 _canceled;
+        [SerializeField] private E1 _waiting;
+        [SerializeField] private E1 _disabled;
+
+        [SerializeField] private V _value;
+        [SerializeField] private BoolVariable _valueAsButton;
```

Comment:
Change to a `BoolReference`

Reply:
That won't really do much... _valueAsButton is only used to set a value that needs to be read from elsewhere, so that crosses out bool or BoolConst. Since CallbackContextInterpreter is a ScriptableObject, that crosses out BoolVariableInstance.

Reply:
Makes sense